HelpSource/Reference/Conditional-Execution.schelp

```diff
@@ -1,32 +1,57 @@
 title::Conditional Execution
 summary:: conditional expressions in SuperCollider
 categories:: Language
-related:: Classes/Boolean,  Classes/Function, Classes/Exception, Reference/Syntax-Shortcuts 
+related:: Classes/Boolean, Classes/Function, Classes/Exception, Reference/Syntax-Shortcuts, Reference/loop
 
-This article gives an overview of the conditional expressions in SuperCollider and the many ways to write them.
-It starts with addressing the relevant syntactic generalities, then discusses the specific conditional expressions in question, 
-and finally touches on optimization issues, the (strongly discouraged) use of if-statements in Synths, and provides a table of classes implementing the methods discussed here.
-For discussions of specific expressions, skip ahead to their respective sections: 
-link::#if::, link::#while::, link::#switch::, link::#case::.
+This page gives an overview of the conditional expressions in SuperCollider and the many ways to write them.
+
+Overview of recommended syntax (more variants exist, see link::#Syntax of conditional expressions#below::):
+table:: 
+## link::#if:: || code:: if(expr) { trueFuncBody } { falseFuncBody } ::
+## link::#case:: || code:: case { testFuncBody1 } { trueFuncBody1 } ... { testFuncBodyN } { trueFuncBodyN } ::
+## link::#switch:: || code:: switch(value) { testValue1 } { trueFuncBody1 } ... { testValueN } { trueFuncBodyN } ::
+## link::#while:: || code:: while { testFuncBody } { loopFuncBody } ::
+::
+
+note::
+To distinguish already instantiated code::Function:: object from the expressions that define them (their "bodies"), the present file adopts the following convention:
+
+code:: someFunc :: emphasis::is equivalent to:: code:: { someFuncBody }:: 
+::
+
+Quick links to sections:
+
+numberedlist:: 
+## link::#Syntax of conditional expressions#syntactic generalities::,
+## link::#if::,  link::#case::, link::#switch::, link::#while:: (specifics),
+## link::#Optimization through inline expansion#optimization/inlining::, 
+## link::#Audio Control Structures: Do not use .if etc. in Synths!#if-statements in Synths:: (strongly discouraged).
+::
+
+For other control structures, see link::Reference/loop:: and link::Classes/Exception::.
 
 section:: Syntax of conditional expressions
 
-Control structures in SuperCollider are implemented as methods called on a receiver; 
-thus, code::if, while,:: etc. are technically emphasis::not:: keywords in SuperCollider but methods,
-even if the syntax can be made to emphasis::look:: as if they were keywords.
+Conditional Expressions in SuperCollider are implemented as strong::methods that are called on a receiver and which return a value::.
+This is distinct from many other languages where, e.g., code::if:: is a reserved identifier (keyword) independent of a particular receiver. 
+For instance, in C or in python, you cannot create a variable called code::if::, whereas in SuperCollider, you can (which does not mean that you should...).
```

Comment:
In SuperCollider all methods are not reserved identifiers. The message `if` is also just a message to a receiver. If this is made clear in the beginning, a lot of other things follow. The whole thing with variable names is confusing, perhaps?

But it is good to clarify all this, anyway.